### DISCUSSION TITLE
Windows 10 Audible App No Longer Available

### DISCUSSION THREAD

**Discussion by mikbe** [audiamus/AaxAudioConverter#117] [Q&A]
Thank you so much for making this app, however, since the Windows 10 Audible app is no longer on the Microsoft store where can I find it? I have books I've paid for and want to listen to them on PC. Thank you again.

**Comment by audiamus** [comment]
Audible has discontinued this app. 

For an alternative, have a look at the new companion to AAX Audio Converter: [Book Lib Connect](https://github.com/audiamus/BookLibConnect), the sister repository right here, next door.

See also #109
